docker/config.properties

```diff
@@ -1,6 +1,8 @@
 inference_address=http://0.0.0.0:8080
 management_address=http://0.0.0.0:8081
 metrics_address=http://0.0.0.0:8082
+grpc_inference_address=0.0.0.0
```

Comment:
how is the port set for now?

Reply:
It uses the default values from here:
https://github.com/pytorch/serve/blob/89c5389d5dba23c32fbed7e7d58234fc30b690a5/frontend/server/src/main/java/org/pytorch/serve/util/ConfigManager.java#L360-L368

And is called here: https://github.com/pytorch/serve/pull/3083/files#diff-45f9e03a9d3046ef0e2c18e18b0bb9c1889f9e3ff9918568870539cc926e3895R454

Reply:
Added configuration option to set grpc address separately from port to maintain backwards compatibility to retain the grpc port configuration:

https://github.com/pytorch/serve/blob/89c5389d5dba23c32fbed7e7d58234fc30b690a5/frontend/server/src/main/java/org/pytorch/serve/util/ConfigManager.java#L104-L105

Reply:
can we keep this consistent?
`grpc_inference_address=http://0.0.0.0:7070`
`grpc_management_address=http://0.0.0.0:7071`

Reply:
It is possible to support configuration format as `grpc_inference_address=http://0.0.0.0:7070` but we've had support to configure the GRPC ports alone since `v0.3.0`:
https://github.com/pytorch/serve/blob/89c5389d5dba23c32fbed7e7d58234fc30b690a5/frontend/server/src/main/java/org/pytorch/serve/util/ConfigManager.java#L104-L105

To maintain backwards compatibility, I believe we'll need to retain the grpc port configuration.

If we were to allow specifying the port along with the address as `grpc_inference_address=http://0.0.0.0:7070`, I believe we'll have two options:
1. Ignore the grpc port configuration ex `grpc_inference_port` if `grpc_inference_address` includes the port already.
2. Override the port specified in `grpc_inference_address` with the port specified `grpc_inference_port`.

Let me know your thoughts or suggestions on better ways to handle it.